backend/src/utils/flattenArray.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Flattens the specified string array.
+ */
+export function flattenArray(array: string[]): string {
```

Comment:
maybe a better name would be `stringifyArray`

Reply:
This is tracked by #235